crates/rspack_core/src/tree_shaking/visitor.rs

```diff
@@ -142,21 +143,81 @@ pub(crate) struct ModuleRefAnalyze<'a> {
   pub(crate) has_side_effects_stmt: bool,
   unresolved_ctxt: SyntaxContext,
   pub(crate) potential_top_mark: HashSet<Mark>,
+  #[allow(dead_code)]
+  comments: Option<&'a SwcComments>,
+}
+
+impl<'a> std::fmt::Debug for ModuleRefAnalyze<'a> {
```

Comment:
I need to implement `Debug` by hand because `SwcComments` doesn't implement `Debug`